lib/ruby-debug-ide/commands/variables.rb

```diff
@@ -129,7 +129,7 @@ def execute
       locals = @state.context.frame_locals(@state.frame_pos)
       _self = @state.context.frame_self(@state.frame_pos)
       begin
-        locals['self'] = _self unless "main" == _self.to_s
+        locals['self'] = _self
```

Comment:
@denofevil I feel this design is OK (and compliant to what's available at runtime), please verify it's OK for you